types/catbox/index.d.ts

```diff
@@ -105,16 +105,16 @@ export interface ClientOptions {
  *  * segment - required when cache is provided. The segment name used to isolate cached items within the cache partition.
  * @see {@link https://github.com/hapijs/catbox#policy}
  */
-export interface Policy extends PolicyAPI {
-    new(options: PolicyOptions, cache: Client, segment: string): Policy;
+export class Policy extends PolicyAPI {
```

Comment:
Except this should be `implements PolicyAPI` and should have implementations for the appropriate methods and members (potentially copies of those in the interface if need be).